packages/editor-ui/src/views/NodeView.vue

```diff
@@ -642,7 +645,7 @@ export default mixins(
 					}
 
 					this.callDebounced('saveCurrentWorkflow', 1000, undefined, true);
-					this.$store.dispatch('settings/fetchPromptsData');
+					this.callDebounced('fetchPromptsData', 1000);
```

Comment:
should not we combine those? same logic that we only fetch if it saves?

Reply:
But saveCurrentWorkflow is somewhere in this mixins. I dont want to mix something there. The safest way is this one :)